lib/systems/inspect.nix

```diff
@@ -49,6 +49,8 @@ rec {
     isM68k         = { cpu = { family = "m68k"; }; };
     isS390         = { cpu = { family = "s390"; }; };
     isS390x        = { cpu = { family = "s390"; bits = 64; }; };
+    isLoongArch    = { cpu = { family = "loongson"; }; };
+    isLoongArch64  = { cpu = { family = "loongson"; bits = 64; }; };
```

Comment:
Why not call the family "loongarch" as well?  "loongson" suggests it would also include e.g. their MIPS processors, which wouldn't be right.

Reply:
Actually, the family as `loongson` is base on it's `proc/cpuinfo` file`(CPU Family : Loongson-64bit)`, I have no ideas about the info with MIPS processors, but I will be trying to connect with the LoongArch engineer about it.

Reply:
I'm sorry, I seem to have misunderstood about your means.